features/aanschrijfwijze.feature

```diff
@@ -93,7 +94,7 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 999992934                       |
       | fields              | naam.aanschrijfwijze.naam       |
```

Comment:
Maak hiervan:

| fields | adressering.aanschrijfwijze.naam |

Dit geldt ook voor de andere 18 occurences in dit bestand waar de 'fields' parameter wordt gespecificeerd.